run_pipeline.py

```diff
@@ -27,13 +27,11 @@ def run_pipeline(config: Config):
     ) = create_dataloaders(config.dataloader_config)
 
     print("| Creating Model...")
-    feature_info = get_feature_info(full_data)
-
     model = Encoder_Decoder_Model(
         hidden_channels=config.hidden_layer_size,
-        feature_info=feature_info,
-        metadata=next(iter(train_loader)).metadata(),
-        embedding=True,
+        feature_info=get_feature_info(full_data),
+        metadata=full_data.metadata(),
+        embedding=False,
```

Comment:
maybe create a ticket about changing this to True later?

Reply:
The error was given by something else, I think embedding works again now. I'm checking.

Reply:
Yes, embedding works.